### ISSUE TITLE
"Copy" Button for Exception Text in the stack trace

### ISSUE THREAD

**Issue by ecxod** [getsentry/sentry#55030]
### Problem Statement

I think it would be nice to have a "Copy" Button for Exception Text in the stack trace area.

Sometimes I just want to copy paste the text of the exception for pasting it in a email or somewhere else. Because of all this restrictions in the browser copy paste became impossible.

### Solution Brainstorm

simply put at the end of the like a "copy" button that loads the text of the exception in the Clipboard.

### Product Area

Issues

**Comment by Plancke** [comment]
Especially since the dropdown menu was introduced, which added another step, it's become more of a hassle. Making the UI remember the raw setting could also be helpful, though a simple copy button as OP suggested is the most likely best approach as it removes basically every hurdle

**Comment by lobsterkatie** [comment]
A version of this request has come up before: https://github.com/getsentry/sentry/issues/56992#issuecomment-1738246087. It's definitely something we would consider doing. Both that ticket and this one are now in our backlog for consideration, and we'll update here once there's any news to share.

**Comment by jjanczyszyn** [comment]
Hi, Wave is very interested in this - has there been any progress on it?

**Comment by jangjodi** [comment]
Hi, @jjanczyszyn - we haven't made any progress on this yet, but it is in our backlog. We will update here when we have any news to share!

**Comment by fokkoru** [comment]
Just want to give a friendly reminder.

**Comment by MichaelSun48** [comment]
Hey all, going to pick this up since it seems like this feature is in high demand. 

@ecxod @fokkoru @jjanczyszyn @eduardb @Plancke 

Let me first clarify what exactly y'all are looking to be easily copy-able

![Image](https://github.com/user-attachments/assets/37263163-c994-4172-a21d-5bc8718e0a65)

Are you referring to the top level exception text (# 1), the code in the stack trace (# 2), or the entire raw stack trace that @malwilley referred to previously?

**Comment by eduardb** [comment]
Hi @MichaelSun48 , that's great news!

I'm personally interested in copying the entire raw stack trace, as then I can paste it in IntelliJ for further analysis. Cheers!

**Comment by eduardb** [comment]
> [@eduardb](https://github.com/eduardb?rgh-link-date=2025-04-07T16%3A51%3A15.000Z) - we have a Command + Option + C keyboard shortcut that copies the event details in Markdown, specifically formatted for LLMs. (we'll be adding a UI entrypoint soon). Does that help with your use case?

@rachrwang No, not at all. My use-case is pasting it into IntelliJ IDEA, but what gets copied with this shortcut is not something that the IDE can interpret, I just tested.

**Comment by scttcper** [comment]
@eduardb is the raw stacktrace something you can paste into intellij idea? I know some stacktraces with threads might still not be formatted correctly for that.

**Comment by eduardb** [comment]
> [@eduardb](https://github.com/eduardb?rgh-link-date=2025-04-07T22%3A04%3A17.000Z) is the raw stacktrace something you can paste into intellij idea? I know some stacktraces with threads might still not be formatted correctly for that.

@scttcper Yes. It works pretty much all the time for me when I manually copy paste it, at least for my use-cases (Android and Python).

The tool I am talking about is called "Analyze Stack Trace or Thread Dump", it's in the menu under "Code", or you can Cmd+Shift+A for it.
